python/lsst/dax/obscore/obscore_exporter.py

```diff
@@ -307,6 +327,119 @@ def to_csv(self, output: str) -> None:
                 for record_batch in self._make_record_batches(self.config.batch_size):
                     writer.write_batch(record_batch)
 
+    def to_votable(self, limit: int | None = None) -> astropy.io.votable.tree.VOTableFile:
+        """Run the export and return the results as a VOTable instance.
+
+        Parameters
+        ----------
+        limit : `int` or `None`, optional
+            Maximum number of records to return. If `None` there is no limit.
+
+        Returns
+        -------
+        votable : `astropy.io.votable.tree.VOTableFile`
+            The resulting matches as a VOTable.
+        """
+        # Get the (possibly cached) ObsCore schema.
+        schema = _get_obscore_schema()
+
+        tables = schema.tables
+        if len(tables) != 1:
+            raise RuntimeError("More than one table defined in ObsCore schema")
+        obscore_columns = {column.name: column for column in tables[0].columns}
+
+        votable = astropy.io.votable.tree.VOTableFile()
+        resource = astropy.io.votable.tree.Resource()
+        votable.resources.append(resource)
+
+        fields = []
+        for arrow_field in self.schema:
+            if arrow_field.name in obscore_columns:
+                ffield = obscore_columns[arrow_field.name]
+                votable_datatype = FelisType.felis_type(ffield.datatype.value).votable_name
+                field = astropy.io.votable.tree.Field(
+                    votable,
+                    name=ffield.name,
+                    datatype=votable_datatype,
+                    arraysize=ffield.votable_arraysize,
+                    unit=ffield.ivoa_unit,
+                    ucd=ffield.ivoa_ucd,
```

Comment:
Could we add the utype here? Pyvo looks for this when getdataurl is called on a record:
https://pyvo.readthedocs.io/en/latest/_modules/pyvo/dal/query.html#Record.getdataurl

```suggestion
                    ucd=ffield.ivoa_ucd,
                    utype=ffield.votable_utype,
```